pegasus/test/test_static_models.rb

```diff
@@ -0,0 +1,23 @@
+require_relative './test_helper'
+require_relative '../../lib/cdo/pegasus'
+require 'timecop'
+
+class StaticModelsTest < Minitest::Test
+  include CaptureQueries
+
+  def test_static_models
+    StaticModels.stubs(:expires_in).returns(2.minutes)
+    Timecop.freeze
+
+    query = -> {PEGASUS_DB[:cdo_donors].all}
+    query.call
+
+    Timecop.travel 1.minute
+    assert_empty(capture_queries(PEGASUS_DB, &query))
+
+    Timecop.travel 2.minutes
+    refute_empty(capture_queries(PEGASUS_DB, &query))
```

Comment:
Oh, so these tests are asserting that no query takes place (because of a cache hit) in the first minute, but a query does take place (because of a cache miss) in the second minute?

Reply:
correct, more or less (I think `travel` might be cumulative so the query takes place after 1 + 2 = 3 minutes, which is well after the 2-minute expiry)